dashboard/app/controllers/robots_controller.rb

```diff
@@ -0,0 +1,27 @@
+class RobotsController < ApplicationController
+  DISALLOWED_ROUTES = [
+    "/admin/",
+    "/api/",
+    "/blockly/",
+    "/dashboardapi/",
+    "/join/",
+    "/milestone/",
+    "/projects/",
+    "/sections/"
+  ].to_set.freeze
+
+  def index
+    if rack_env == :production
```

Comment:
Did you test this render locally?

Reply:
It renders locally with the disallow. Is there a way to pretend I'm on production locally so I can check the different list for that?

Reply:
Not easily. I'd recommend temporarily inverting the conditional locally so you can test the render in development

Reply:
I'd just change this if statement to `if true` and make sure it looks right (then obviously throw away that update) 🤷‍♀️

Reply:
Tested and works!